apps/bors_frontend/web/views/layout_view.ex

```diff
@@ -7,19 +7,27 @@ defmodule BorsNG.LayoutView do
   use BorsNG.Web, :view
 
   def get_version do
-    hash = get_heroku_commit() || get_git_commit() || get_release_version()
-    {String.slice(hash, 0..6), hash}
+    get_commit() || get_tag()
   end
 
-  def get_heroku_commit do
-    "HEROKU_SLUG_COMMIT"
-    |> System.get_env()
-    |> case do
-      nil -> nil
-      commit -> String.slice(commit, 0..10)
```

Comment:
why it changed from `0..10` to `0..6`?

Reply:
That's how many characters GitHub itself shows.